examples/creatives/nonlinear_overlay.js

```diff
@@ -6,4 +6,20 @@ class NonLinear extends BaseSimidCreative {
     constructor() {
       super();
     }
+
+    onRequestResize() {
+      const resize_x_val = document.getElementById('resize_x_val').value;
```

Comment:
optional nit. if you name all of these the same thing as the key they will be used for below, you can define the object using shorthand properties

```
const x_val = element.value;
const params = {x_val};
```

Reply:
took Ryan's idea and just set the properties of the original empty object